.github/workflows/add_label.yml

```diff
@@ -0,0 +1,41 @@
+name: Add "ready-to-merge" label
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  add_label:
+    name: Add label
+    if: startsWith(github.event.comment.body, 'bors r+') || contains(toJSON(github.event.comment.body), '\r\nbors r+') || startsWith(github.event.comment.body, 'bors merge') || contains(toJSON(github.event.comment.body), '\r\nbors merge')
```

Comment:
This looks silly, but I couldn't figure out a better way to detect `bors r+` at the start of a line. [Related question I asked](https://github.community/t5/GitHub-Actions/Searching-for-a-substring-with-a-newline-using-the-quot-contains/m-p/53201#M8740).

Reply:
Call me old-fashioned, but I'd write all of that in a shell script.  The actions language is clearly inferior to bash for real programs.

Reply:
Yeah, the reason I did it this way is that if this condition is false, we can avoid starting the job altogether.

Reply:
Oh, I see.  Then it doesn't even show up in the status list?  That's good.

Reply:
Only the `add_comment.yml` workflow can show up in the status list on the PR, and indeed, if that one is triggered by the addition of a label other "ready-to-merge" so that the job isn't started, it won't show up in the status.